Linux_cmake/nheqminer_cpu_tromp/CMakeLists.txt

```diff
@@ -0,0 +1,143 @@
+project(nheqminer_cpu_tromp)
+cmake_minimum_required(VERSION 2.8)
+
+#aux_source_directory(. SRC_LIST)
+#add_executable(${PROJECT_NAME} ${SRC_LIST})
+
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
```

Comment:
I know this is a WIP, but this cmake is messy/a duplicate. Can we create a single (multiplatform) `CMakeLists` and use `ADD_SUBDIRECTORY` to add cmake files for specific miners? (tromp, xenoncat, cuda, ...)